src/components/OnlyOn/OnlyOn.js

```diff
@@ -22,10 +22,10 @@ class OnlyOn extends PureComponent<Props> {
         return LessThanSmall;
       case 'md':
         return LessThanMedium;
-      case 'smMin':
-        return SmallAndUp;
       case 'mdMin':
         return MediumAndUp;
+      case 'lgMin':
+        return LargeAndUp;
```

Comment:
I'm not sure about this change. But `smMin` didn't exist so I'm assuming this was the intent